src/Components/ProductFilter/ProductFilter.js

```diff
@@ -19,19 +25,31 @@ const ProductFilter = () => {
     );
   }
 
+  function resetFilters() {
+    setFilterState([]);
+  }
+
   return (
     <div className="ProductFilter">
       <h2>Categories</h2>
       <ul>
-        {categories.map((category) => (
+        {data.results.map((category) => (
           <label
             key={category.id}
             onClick={() => onCheckboxClick(category.id)}
             className={isSelected(category.id) ? 'active' : ''}
+            id={category.id}
           >
             {category.data.name}
           </label>
         ))}
+        {filterState.length > 0 && (
+          <div className="button-container">
+            <button className="simple-button" onClick={() => resetFilters()}>
```

Comment:
Creo que podrías ahorrar declarar el metodo porque estas haciendo solo el setState
```suggestion
            <button className="simple-button" onClick={() => setFilterState([])}>
```